From 44514f7dba1dac1cc6ba3be0039ba36a180ac0b1 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Sun, 27 Dec 2020 16:35:33 +0100
Subject: [PATCH] treewide: move /var/run/ubus.sock to /var/run/ubus/ubus.sock
In upstream, there is a idea to run ubusd as non-root user.
Because of that, they moved /var/run/ubus.sock to
/var/run/ubus/ubus.sock

---
 utils/turris-maintain/Makefile                  | 2 +-
 utils/turris-maintain/files/lighttpd-restart.py | 2 +-
 utils/turris-maintain/files/network-restart.py  | 2 +-
 utils/turris-maintain/files/reboot.py           | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/utils/turris-maintain/Makefile b/utils/turris-maintain/Makefile
index b2f8bd2fd..3ac9944b9 100644
--- a/utils/turris-maintain/Makefile
+++ b/utils/turris-maintain/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk

 PKG_NAME:=turris-maintain
 PKG_VERSION:=11
-PKG_RELEASE:=1
+PKG_RELEASE:=2

 PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>

diff --git a/utils/turris-maintain/files/lighttpd-restart.py b/utils/turris-maintain/files/lighttpd-restart.py
index d0d6c99db..3ecb31606 100644
--- a/utils/turris-maintain/files/lighttpd-restart.py
+++ b/utils/turris-maintain/files/lighttpd-restart.py
@@ -25,7 +25,7 @@ bus = uci_get("foris-controller", "main", "bus", "ubus")
 controller_id = None

 if bus == "ubus":
-    path = uci_get("foris-controller", "ubus", "notification_path", "/var/run/ubus.sock")
+    path = uci_get("foris-controller", "ubus", "notification_path", "/var/run/ubus/ubus.sock")
     from foris_controller.buses.ubus import UbusNotificationSender
     sender = UbusNotificationSender(path)
 elif bus == "unix":
diff --git a/utils/turris-maintain/files/network-restart.py b/utils/turris-maintain/files/network-restart.py
index 2b2ce3c18..56749c11a 100644
--- a/utils/turris-maintain/files/network-restart.py
+++ b/utils/turris-maintain/files/network-restart.py
@@ -26,7 +26,7 @@ bus = uci_get("foris-controller", "main", "bus", "ubus")
 controller_id = None

 if bus == "ubus":
-    path = uci_get("foris-controller", "ubus", "notification_path", "/var/run/ubus.sock")
+    path = uci_get("foris-controller", "ubus", "notification_path", "/var/run/ubus/ubus.sock")
     from foris_controller.buses.ubus import UbusNotificationSender
     sender = UbusNotificationSender(path)
 elif bus == "unix":
diff --git a/utils/turris-maintain/files/reboot.py b/utils/turris-maintain/files/reboot.py
index b49f9f017..9583da318 100644
--- a/utils/turris-maintain/files/reboot.py
+++ b/utils/turris-maintain/files/reboot.py
@@ -18,7 +18,7 @@ with euci.EUci() as uci:
     if bus == "ubus":
         path = uci.get(
             "foris-controller", "ubus", "notification_path",
-            default="/var/run/ubus.sock"
+            default="/var/run/ubus/ubus.sock"
         )
         from foris_controller.buses.ubus import UbusNotificationSender
         sender = UbusNotificationSender(path)
--
2.25.1
